packages/hardhat-zksync-solc/src/index.ts

```diff
@@ -169,13 +171,13 @@ subtask(TASK_UPDATE_SOLIDITY_COMPILERS, async (_args: any, hre: HardhatRuntimeEn
         .find((extractor) => extractor.suitable(userSolidityConfig))
         ?.extract(userSolidityConfig);
 
-    const latestEraVersion = await getLatestEraVersion();
+    // const latestEraVersion = await getLatestEraVersion();
 
     hre.config.solidity.compilers.forEach(async (compiler) =>
         updateBreakableCompilerConfig(
             { compiler },
             hre.config.zksolc,
-            latestEraVersion,
+            semver.gte(hre.config.zksolc.version, '1.5.12') ? '1.0.2' : '1.0.1',
```

Comment:
Is the `greater-than-or-equal`?
`1.0.2` must be selected strictly for `1.5.13` or higher.